proto/stat_var.proto

```diff
@@ -181,10 +181,19 @@ message SearchStatVarRequest {
   bool enable_blocklist = 3;
 }
 message SearchStatVarResponse {
```

Comment:
Nit: can move SearchResultSVG one level up, so it's easier to reference in the code

Reply:
done